src/react-native-animated-charts/src/charts/linear/ChartPath.tsx

```diff
@@ -0,0 +1,380 @@
+          if (typeof i === 'undefined' || values.x === null) {
+            return 0;
+          }
+
+          return Math.hypot(currentPath.points[i].x - Math.floor(values.x));
+        });
+
+        setOriginData(currentPath, index);
+      },
+      [currentPath]
+    );
+
+    const animatedProps = useAnimatedProps(() => {
+      const props: PathProps & ViewProps = {};
+
+      if (!currentPath) {
+        return {
+          d: '',
+        };
+      }
+
+      props.d = interpolatorWorklet().value
+        ? interpolatorWorklet().value(progress.value)
+        : currentPath.path;
+
+      props.strokeWidth =
+        pathOpacity.value *
+          (Number(strokeWidth) - Number(selectedStrokeWidth)) +
+        Number(selectedStrokeWidth);
+
+      if (Platform.OS === 'ios') {
+        props.style = {
+          opacity: pathOpacity.value * (1 - selectedOpacity) + selectedOpacity,
+        };
+      }
+
+      return props;
+    }, [currentPath]);
+
+    const onGestureEvent = useAnimatedGestureHandler<LongPressGestureHandlerGestureEvent>(
+      {
+        onActive: event => {
+          if (!isActive.value) {
+            isActive.value = true;
+
+            pathOpacity.value = withTiming(
+              0,
+              timingFeedbackConfig || timingFeedbackDefaultConfig
+            );
+
+            if (hapticsEnabled) {
+              impactHeavy();
+            }
+          }
+
+          state.value = event.state;
+          translationX.value = positionXWithMargin(event.x, hitSlop, width);
+          translationY.value = event.y;
+        },
+        onCancel: event => {
+          state.value = event.state;
+          resetGestureState();
+        },
+        onEnd: event => {
+          state.value = event.state;
+          resetGestureState();
+
+          if (hapticsEnabled) {
+            impactHeavy();
+          }
+        },
+        onFail: event => {
+          state.value = event.state;
+          resetGestureState();
+        },
+        onStart: event => {
+          // WARNING: the following code does not run on using iOS, but it does on Android.
+          // I use the same code from onActive
+          // platform is for safety
+          if (Platform.OS === 'android') {
+            state.value = event.state;
+            isActive.value = true;
+            pathOpacity.value = withTiming(
+              0,
+              timingFeedbackConfig || timingFeedbackDefaultConfig
+            );
+
+            if (hapticsEnabled) {
+              impactHeavy();
+            }
+          }
+        },
+      },
+      [width, height, hapticsEnabled, hitSlop, timingFeedbackConfig]
+    );
+
+    const pathAnimatedStyles = useAnimatedStyle(() => {
+      return {
+        opacity: pathOpacity.value * (1 - selectedOpacity) + selectedOpacity,
+      };
+    }, []);
+
+    return (
+      <View style={{ height, width }}>
+        <LongPressGestureHandler
+          {...{ onGestureEvent }}
```

Comment:
```suggestion
          {...{ onGestureEvent }}
          onGestureEvent={onGestureEvent}
```

Reply:
why did this get resolved?

Reply:
I actually changed this already but seems like it's got missing